### PR TITLE
Features/filter product by price

### PR DESCRIPTION
**Pull request by Datto098**

### CODE REVIEW

**Comment by Datto098** [comment]
Update project directory, Update filter product by discount price

File: backend/src/main/java/runtrail/dev/backend/controller/SpuController.java
Comment (10102004tan) [inline backend/src/main/java/runtrail/dev/backend/controller/SpuController.java:1]: cho ví dụ tại sao ở đây giá dùng double

Comment (Datto098) [reply backend/src/main/java/runtrail/dev/backend/controller/SpuController.java:1]: Nảy test cái làm tròn giá nên chuyển qua Double thử =)), [Đã sửa lại]

**Review by tranhieuphuc12 — DISMISSED** [review]
no issue, be able to merge

**Review by Charcasbeos — CHANGES_REQUESTED** [review]
When I enter the maximum price with a 20-digit number, the interface overflow

**Comment by Datto098** [comment]
> When I enter the maximum price with a 20-digit number, the interface overflow

Updated

**Review by chiendevj — DISMISSED** [review]
successfully to do.

**Review by DoNgocHieu — APPROVED** [review]
oke success , done

**Review by khang3999 — APPROVED** [review]
Great

**Review by Charcasbeos — APPROVED** [review]
It's oke

**Pull request merged**